src/main/java/org/cactoos/func/FuncOf.java

```diff
@@ -95,7 +95,7 @@ public FuncOf(final Proc<X> proc, final Y result) {
      * Ctor.
      * @param fnc Func
      */
-    public FuncOf(final Func<X, Y> fnc) {
+    private FuncOf(final Func<X, Y> fnc) {
```

Comment:
@yegor256 @fabriciofx Yegor is not against private constructors instead he against multiple independent constructors.